dotcom-rendering/scripts/webpack/browser.js

```diff
@@ -66,48 +72,30 @@ module.exports = ({ isLegacyJS }) => ({
 		rules: [
 			{
 				test: /\.[jt]sx?|mjs$/,
+				loader: 'esbuild-loader',
```

Comment:
Minor but I don't think `esbuild-loader` has been added to the dependencies?
It must be coming in through another dependency - storybook I think.
It's a non-issue for this spike but it might cause some head-scratching if you're expecting features or perf benefits from the latest version.